web/src/main/java/org/wso2/testgrid/web/sso/SSOSessionCheckFilter.java

```diff
@@ -68,6 +68,16 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
                 String path = ((HttpServletRequest) servletRequest).getRequestURI();
                 if (isSecuredAPI(path)) {
                     Boolean isSessionValid = ((HttpServletRequest) servletRequest).isRequestedSessionIdValid();
+
+                    // Adding a token based Authentication to call backend api without SSO for testing.
+                    if (!StringUtil.isStringNullOrEmpty(
+                            ((HttpServletRequest) servletRequest).getHeader("Authorization"))) {
+                        if (((HttpServletRequest) servletRequest).getHeader("Authorization").equals
+                                (ConfigurationContext.getProperty(ConfigurationProperties.API_TOKEN))) {
```

Comment:
Just know that 'Authorization' header has a specific format - `Authorization: <type> <credentials>`

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization#Syntax